service/controller/resource/release/delete.go

```diff
@@ -0,0 +1,9 @@
+package accountid
+
+import (
+	"context"
+)
+
+func (r *Resource) EnsureDeleted(ctx context.Context, obj interface{}) error {
+	return nil
```

Comment:
The `accountid` resource adds the value to the context in `EnsureDeleted` as well. That doesn't seem necessary here, but please let me know if I'm misunderstanding.